tests/integration/test_data/gomod_e2e_multiple_modules/bom.json

```diff
@@ -0,0 +1,215 @@
+{
+  "bom_format": "CycloneDX",
```

Comment:
This must be `bomFormat` per the JSON schema.

Can we, for each test, check that our CycloneDX is valid?

Reply:
Good catch. I was intending to actually move all keys from snake_case to camelCase at some point, but I forgot about it.

I will write a simple validator to avoid pulling in the entire CycloneDX lib. Wdyt?

Reply:
We can validate it with `jsonschema` like Robert did in Cachito